app/services/notify_app_store.rb

```diff
@@ -8,7 +8,9 @@ def self.process(submission:)
     else
       begin
         new.notify(MessageBuilder.new(submission:))
-        SubmissionFeedbackMailer.notify(submission).deliver_later!
+
+        klass = "#{submission.namespace}::SubmissionFeedbackMailer".constantize
```

Comment:
just as a note an alternative to `constantize`

```
submission.namespace.const_get('SubmissionFeedbackMailer')
```

Reply:
cool, using this just for cosistency with in `app/view_models/base_view_model.rb`.